doc/devel/development_setup.rst

```diff
@@ -82,6 +80,10 @@ so that you can access the latest changes there (see also `Managing remote repos
 
          git remote add upstream git@github.com:matplotlib/matplotlib.git
 
+You can now use ``upstream`` to retrieve the most recent merged changes to the
+Matplotlib source code (see also `Managing remote repositories
+<https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories>`__.)
```

Comment:
```suggestion
You can then retrieve the current development state of the Matplotlib source code via ::

    git checkout main
    git pull upstream main

```
We should state this explicitly. The mentioned link does not cover this.

Reply:
Not sure what you don’t love about the elsewhere advice, but I just opened #25857 and would love you to weigh in there if it’s relevant.